corehq/apps/reports/static/reports/js/edit_submission.js

```diff
@@ -1,7 +1,6 @@
 hqDefine("reports/js/edit_submission", function () {
     $(function () {
         var initialPageData = hqImport("hqwebapp/js/initial_page_data").get;
-        window.MAPBOX_ACCESS_TOKEN = initialPageData('mapbox_access_token'); // maps api is a global variable depended on by touchforms
```

Comment:
I know the mapbox api token is used somewhere in reports, are you sure removing this doesn't affect it?

Reply:
This isn't a report (despite the name). It's the "edit forms" feature, which lets you take an existing form submission and update it by loading that form into the web apps UI. Ultimately this page includes all of web apps's javascript.